workflows/codefresh-csdp/versions/1.1.0/images/image-enricher-git-info/src/configuration/index.js

```diff
@@ -54,6 +60,20 @@ const schema = Joi.object({
         GITLAB_HOST_URL: Joi.string().uri().empty(''),
         GITLAB_TOKEN: Joi.string().required()
     })
+})
+.when(Joi.object({ GIT_PROVIDER: Joi.valid(providers.BITBUCKET_SERVER) }).unknown(), {
```

Comment:
Can be changed to single `when` expression:
```suggestion
.when(Joi.object({ GIT_PROVIDER: Joi.valid(providers.BITBUCKET, providers.BITBUCKET_SERVER) }).unknown(), {
```